packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1081,11 +1090,11 @@ export function createProjectionNode<I>({
         /**
          * Animation
          */
-        animationProgress = 0
```

Comment:
What was this used for before?

Reply:
Nothing, must have been left around from long ago